modules/terraform/init_test.go

```diff
@@ -82,3 +82,71 @@ func TestInitPluginDir(t *testing.T) {
 
 	assert.Contains(t, initOutput, "(unauthenticated)")
 }
+
+func TestInitReconfigureBackend(t *testing.T) {
+	t.Parallel()
+
+	stateDirectory, err := ioutil.TempDir("", t.Name())
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
NIT: 

```suggestion
	require.NoError(t, err)
```

Reply:
Same applies to other `if err != nil { t.Fatal(err) }` blocks below.

Reply:
fixed according your recommendation